packages/react/.storybook/main.ts

```diff
@@ -4,19 +4,17 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+import fs from 'fs';
```

Comment:
I think it would be a good practice to put `node:` before using each node core modules/lib, etc.

example:

`import fs from 'node:fs'` [fs Docs](https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#file-system)
`import path from 'node:path'` [Path docs](https://nodejs.org/dist/latest-v16.x/docs/api/path.html#path)

Reply:
oh cool I didnt know you could reference these like this. good to know!